cluster/definition.go

```diff
@@ -140,6 +140,11 @@ func (d Definition) NodeIdx(pID peer.ID) (NodeIdx, error) {
 
 // VerifySignatures returns true if all config signatures are fully populated and valid. A verified definition is ready for use in DKG.
 func (d Definition) VerifySignatures() error {
+	// Skip signature verification for definition versions earlier than v1.3 since there are no EIP712 signatures before v1.3.0.
+	if !supportEIP712Sigs(d.Version) {
+		return nil
```

Comment:
add `TODO: maybe error if signatures present`

Reply:
we error in UnmarshalJSON though

Reply:
sure, but we should also maybe error here. If I create a old version with signatures, it shouldn't be valid. On that note, we should also error when marshalling probably.

Reply:
done